### PR TITLE
Improve default network offering setting for egress policy

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description

This PR addresses #5957

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

### How Has This Been Tested?
1. Create an Isolated Network offering setting the default egress policy (if firewall service is checked) - true / false
2. Create a network using that offering
3. Notice in the details tab of the created network you will find details of the policy:
image
4. Go to the egress rules tab, there's a note for user to understand the implication of the policy / rules they add:
image

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan test

File: ui/src/config/section/network.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default {
         return fields
       },
       details: () => {
-        var fields = ['name', 'id', 'description', 'type', 'traffictype', 'vpcid', 'vlan', 'broadcasturi', 'cidr', 'ip6cidr', 'netmask', 'gateway', 'aclname', 'ispersistent', 'restartrequired', 'reservediprange', 'redundantrouter', 'networkdomain', 'zonename', 'account', 'domain']
```
Comment (weizhouapache) [inline ui/src/config/section/network.js:41]: @Pearl1594 
would it be good to highlight the egress policy or the behavior (allow/deny) of egress rules in "Egress rules" tab ?
e.g.  if egress policy is true,  display a message like "The traffic matching the following rules will be denied"

File: ui/src/views/offering/AddNetworkOffering.vue [hunk 2]
```diff
@@ -342,7 +342,7 @@
           <tooltip-label slot="label" :title="$t('label.egressdefaultpolicy')" :tooltip="apiParams.egressdefaultpolicy.description"/>
           <a-radio-group
             v-decorator="['egressdefaultpolicy', {
-              initialValue: 'allow'
+              initialValue: 'deny'
```
[the remark this answers is not in the document]
Comment (Pearl1594) [reply ui/src/views/offering/AddNetworkOffering.vue:345]: @weizhouapache I made this change to align with the following comment on #5957 

> Ensure that the default value of the Egress Policy switch is set to Deny for the Isolated Network Service Offering

File: ui/src/config/section/network.js (hunk 1 above)
Comment (Pearl1594) [reply ui/src/config/section/network.js:41]: Sure @weizhouapache , I can make the necessary changes to highlight was is implied when egress policy is true / false. Thanks

**Review by nvazquez — APPROVED** [review]
Code LGTM

File: ui/src/views/offering/AddNetworkOffering.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (weizhouapache) [reply ui/src/views/offering/AddNetworkOffering.vue:345]: @Pearl1594 ok. fine with me.

**Review by DaanHoogland — COMMENTED** [review]
clgtm,
can you expand on testing for this @Pearl1594

**Comment by Pearl1594** [comment]
> clgtm, can you expand on testing for this @Pearl1594

@DaanHoogland Sorry for the delay in response.
So to test this - 
1. Create an Isolated Network offering setting the default egress policy (if firewall service is checked) - true / false
2. Create a network using that offering
3. Notice in the details tab of the created network you will find details of the policy:
image
4. Go to the egress rules tab, there's a note for user to understand the implication of the policy / rules they add:
image

cc @weizhouapache I've address your comments. Thanks for the review.

**Comment by Pearl1594** [comment]
@blueorangutan package

**Comment by Pearl1594** [comment]
@blueorangutan ui

**Comment by DaanHoogland** [comment]
> > clgtm, can you expand on testing for this @Pearl1594
> 
> @DaanHoogland Sorry for the delay in response. So to test this -
> 
>     1. Create an Isolated Network offering setting the default egress policy (if firewall service is checked) - true / false
> 
>     2. Create a network using that offering
> 
>     3. Notice in the details tab of the created network you will find details of the policy:
>        image
> 
>     4. Go to the egress rules tab, there's a note for user to understand the implication of the policy / rules they add:
>        image

I meant for the test description to go into the PR description, @Pearl1594

**Comment by Pearl1594** [comment]
done @DaanHoogland

**Comment by Pearl1594** [comment]
@blueorangutan test

**Review by DaanHoogland — APPROVED** [review]

**Review by weizhouapache — APPROVED** [review]
everything looks good.
tested ok.

**Pull request merged**